### DISCUSSION TITLE
Table scrollTo()功能，virtual=false， 非虚拟表格跳转指定行 还会导致页面滚动

### DISCUSSION THREAD

**Discussion by xiaobing0723** [ant-design/ant-design#51651] [Q&A]
Table scrollTo()功能，非虚拟表格跳转指定行 还会导致页面滚动，官网示例：https://stackblitz.com/edit/react-ychcvz?file=demo.tsx

**Comment by petercat-assistant** [comment]
在 Ant Design 的 GitHub 仓库中，我找到了一个相关的问题 [Issue #28647 - Scroll to row of virtual table](https://github.com/ant-design/ant-design/issues/28647)。虽然这个问题是关于虚拟表格的，但可能会提供一些关于 `scrollTo()` 方法的实现和行为的见解。

如果您遇到的问题与此不同，或者该问题没有解决您的问题，您可以考虑在 GitHub 上提交一个新的问题报告，以便开发团队能够进一步调查和解决。

如果您有其他问题或需要进一步的帮助，请随时告诉我。

如需进一步帮助，请在评论中描述您的问题并@petercat-assistant与我开始对话。
